etc/adminhtml/system/debug.xml

```diff
@@ -7,23 +7,23 @@
 -->
 <include xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_include.xsd">
-    <group id="debug" translate="label" sortOrder="300" showInDefault="1" showInWebsite="0" showInStore="0">
+    <group id="debug" translate="label" sortOrder="300" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
I think we should leave the logging options to only be available in the default scope as the log file is global and we also process hooks outside of store/website scope.

Reply:
updated